pages/instructor-checklist.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: page-fullwidth
+show_meta: false
+title: "Instructor checklist"
+header:
+   image_fullwidth: "wood_plank.jpg"
+permalink: "/instructor-checklist/"
+---
+
+Each instructor is responsible for preparing to teach their section(s) of the workshop. Remember that our lessons are continuously evolving, so even if you have taught the content before, you will need to check the materials for changes.
+
+## Before the workshop:  
+- [ ] Coordinate with the lead instructor to decide who is teaching what, and in which order  
+- [ ] If your workshop is self-organized, meet with the lead instructor to be trained in [Data Carpentry’s teaching practices](https://swcarpentry.github.io/instructor-training/16-practices/)  
+- [ ] Test your installation instructions to make sure they work as expected  
+- [ ] Read the [lesson(s)](http://www.datacarpentry.org/lessons/) you are going to teach in detail  
+- [ ] If there is an [instructor’s guide](/for-instructors/#contribute-to-instructor-notes) for your lesson, read it carefully  
+- [ ] Do a dry-run of your entire lesson  
+- [ ] Read through our [troubleshooting document](/troubleshooting/)  
+
+## At the workshop:  
+- [ ] Remind your learners about the [Code of Conduct](http://www.datacarpentry.org/code-of-conduct/)  
+- [ ] Remind learners to use their sticky notes to give feedback - see description of how these are used [here](http://swcarpentry.github.io/instructor-training/16-practices/)  
+- [ ] Get feedback with minute cards at lunch and end of each day - see description of minute cards [here](http://swcarpentry.github.io/instructor-training/16-practices/)
+
+## After the workshop:  
+- [ ] Take part in an [instructor discussion session](http://pad.software-carpentry.org/instructor-discussion)  
+- [ ] Send any feedback you have to [Data Carpentry admin](mailto:admin@datacarpentry.org) (admin@datacarpentry.org)
```

Comment:
I assume lead instructor is responsible for sending attendance counts, so this instruction is currently only on the lead instructor checklist.